VultisigApp/VultisigApp/Views/Keygen/FastVaultEnterPasswordView.swift

```diff
@@ -19,13 +19,19 @@ struct FastVaultEnterPasswordView: View {
     let vault: Vault
     let onSubmit: (() -> Void)?
 
+    private let keychain = DefaultKeychainService.shared
+    private let biometryService = BiometryService.shared
+
     var view: some View {
         VStack {
             passwordField
             Spacer(minLength: 20)
             disclaimer
             buttons
         }
+        .onAppear {
+            tryAuthenticate()
```

Comment:
1. Not everyone want to save their password , so we have to provide an option , a checkbox on the screen , when user tick it , after they put in their password for the first time  then we save the password to keychain , otherwise keep as it is
2. we need to save the information somewhere , if user didn't save their password to keychain , then don't request user to authenticate

Reply:
<img width="194" alt="image" src="https://github.com/user-attachments/assets/ee4ce258-b8fe-47f6-a052-c7e70fe60f1d">
<img width="193" alt="image" src="https://github.com/user-attachments/assets/9cb3af9b-0c46-4742-8e75-b80d284b68e3">

Reply:
I think here , the issue still has not been addressed , like if I didn't enable Biometrics , then I should not been prompt about Permission to access keychain 

If I already enable BioMetrics , then we should prompt permission to access keychain , but no password dialog is needed